modin/data_management/dispatcher.py

```diff
@@ -0,0 +1,167 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+import os
+
+from modin import __execution_engine__ as execution_engine
+from modin import __partition_format__ as partition_format
+
+from modin.data_management import factories
+
+
+class FactoryNotFoundError(AttributeError):
+    def __init__(self, partition_format, execution_engine):
+        super().__init__(
+            "Cannot find a factory for partition '{}' and execution engine '{}'. Potential reason might be incorrect environment variable value for MODIN_BACKEND or MODIN_ENGINE".format(
```

Comment:
Can you split string? Something like:

```suggestion
            "Cannot find a factory for partition '{}' and execution engine '{}'."
            " Potential reason might be incorrect environment variable value for MODIN_BACKEND or MODIN_ENGINE".format(
```

Reply:
took @devin-petersohn suggestion